### DISCUSSION TITLE
L-BFGS optimizer with fractional diffusion equation

### DISCUSSION THREAD

**Discussion by ginasn** [lululxvi/deepxde#1161] [Q&A]
Hi everyone,

I am working on the space-fractional diffusion equation using the given pinn foward example in the deepxde. I am trying to train using adam as in the code but then train the model using L-BFGS. However it is failing. Below is the error. 
When I compile with Adam it works fine. I am relatively new to deep learning and want to know if it would be possible to use the L-BFGS for this equation. Thank you for helping me with this!
![adam_fpde](https://user-images.githubusercontent.com/125862557/221396186-bfc5eec5-851a-4584-aafc-cf337b661e6e.png)
![lbfgs_fpde](https://user-images.githubusercontent.com/125862557/221396188-32bb8d5c-9cbf-4404-96d2-ed4411fe387b.png)
![lbfgs_fpde2](https://user-images.githubusercontent.com/125862557/221396314-bcd62b5e-f2a0-4a0e-8f35-c5c0c2bce838.png)

**Comment by lululxvi** [comment]
Interesting... Not sure why.

**Reply by ginasn** [reply]
Will there be a fix for this? It would be very helpful.

**Comment by lululxvi** [comment]
You may try the backend paddle.
